BackEndFlask/Functions/genericImport.py

```diff
@@ -0,0 +1,124 @@
+from typing import List
+
+from core import db
+from Functions.test_files.population_functions import *
+from Functions.helper import helper_verify_email_syntax, helper_create_user, helper_ok, helper_cleanup
+from Functions.customExceptions import *
+from models.user import *
+from models.role import get_role  # used for getting role id from string role
+from models.user_course import *
+from sqlalchemy import *
+import itertools
+import csv
+
+def genericcsv_to_db(user_file: str, owner_id: int, course_id: int) -> None|str:
+    """ 
+    DESCRIPTION:
+    Takes a csv file and creates users of any type (student, TA, etc.)
+    and adds them to the database.
+
+    PARAMETERS:
+    user_file: str: The path to the csv file.
+    owner_id: int:  The user_id of the owner of the course.
```

Comment:
I believe you could remove the owner_id parameter and instead get it from the database by finding the owner_id of the course with id course_id. Not a huge deal but would simplify calling the function and potentially calling it without matching course_id and owner_id.